### PR TITLE
Bug - Printing wrong config & Logging info when warning

### PR DESCRIPTION
**Pull request by GodVenn**
The printed config did not show which robot package was being used, and also did not include any module names, leaving the user guessing what the default value was.
The logger logged failed connect attempts as info when it should be warning

### CODE REVIEW

File: src/isar/modules.py
Comment (aestene) [inline src/isar/modules.py:182]: What is the reasoning behind moving these variables outside the function itself? This way they will always be in memory as "global" variables in this file while I'm not certain they need to be?

**Review by aestene — CHANGES_REQUESTED** [review]
Have a look at the one comment.

Otherwise everything looks good, good improvements!

File: src/isar/modules.py
Comment (GodVenn) [reply src/isar/modules.py:182]: It was so that the _append_key function would have access to them as if they where all inside a static class. 
The alternative is to pass them to each call to that function.
I'm not that comfortable with python yet, so if you think that's a better alternative then let's do that!

Comment (aestene) [reply src/isar/modules.py:182]: @Christdej @tsundvoll 

Any thoughts on this? Personally something feels off to me about having them as open variables within the file in this manner as they are then exported and it's possible to import them. But I'm not certain if it's a bad or good pattern.

Comment (tsundvoll) [reply src/isar/modules.py:182]: I agree that it looks a bit off. It can be a symptom of the entire file being written a bit hard to read. I am looking into a better way to formulate this now.

Comment (GodVenn) [reply src/isar/modules.py:182]: Maybe moving the functionality of this last part of the file to its own file could be a solution?

Comment (aestene) [reply src/isar/modules.py:182]: It could be moved but then again it is very tightly coupled with the modules defined in this file.

I think I'll say approved from my side, but let @tsundvoll comment before you merge to see if he thought of a smooth solution.

**Review by aestene — APPROVED** [review]
LGTM

**Review by Christdej — CHANGES_REQUESTED** [review]
Me and Thomas discussed this yday, dont merge yet

File: src/isar/modules.py
Comment (tsundvoll) [reply src/isar/modules.py]: We found a smooth solution that resolves this comment.

**Review by Christdej — APPROVED** [review]

**Pull request merged**
